src/components/PageElements/Elements/ReviewApplicantResponse.tsx

```diff
@@ -49,16 +65,26 @@ const ReviewApplicantResponse: React.FC<ReviewApplicantResponseProps> = ({
               reviewResponse as ReviewResponse /* Casting to ReviewResponse since decision would exist if reviewResponse is defined */
             }
           >
-            {isActiveReviewResponse && <UpdateIcon onClick={showModal} />}
+            {getReviewDecisionOption()}
           </ReviewResponseElement>
           {/* div below forced border on review response to be square */}
           <div />
+          {/* Consolidation Response */}
+          {consolidationReviewResponse && (
+            <ReviewResponseElement
+              isCurrentReview={false}
```

Comment:
would be good to indicate in review response that it's been updated `<UpdatedLabel />`

Reply:
Hmmm I'll add this to the list of things to check with Debbie. It seems this makes more sense from the Applicant's perspective, since we show only in the Summary page at last so they can see all changes. In the Review page, you they can see all their current reviews with the "edit" icon or Update button - when changes are required. Not sure if would be better to display something extra about changes done in this review... Let's see.

Reply:
Ok, i thought it would be the same as change request to applicant (show updated indicator when it's been updated, with edit button).